### PR TITLE
Update Zonal stats

### LINKED ISSUE #587: Allow selection of wanted stats for zonal stats (fix #4429)
https://github.com/qgis/QGIS/commit/e01dca759d2b69bf9749b7855c83ec5832d427e1
Author: Nyall Dawson
Date:   Mon Apr 20 18:19:52 2015 +1000

### PR DESCRIPTION
**Pull request by DelazJ**
fixes #587, #588, #589

### CODE REVIEW

**Comment by yjacolin** [comment]
Looks good to me. Thanks.

File: source/docs/user_manual/plugins/plugins_zonal_statistics.rst [hunk 1]
```diff
@@ -8,9 +8,21 @@ Zonal Statistics Plugin
 With the |icon_zonal_statistics| :guilabel:`Zonal statistics` plugin, you can analyze
 the results of a thematic classification. It allows you to calculate several values
 of the pixels of a raster layer with the help of a polygonal vector layer (see
-figure_zonal_statistics_). You can calculate the sum, the mean value and the
-total count of the pixels that are within a polygon. The plugin generates output
-columns in the vector layer with a user-defined prefix.
+figure_zonal_statistics_). Choosing a color band, the plugin generates output
+columns in the vector layer with a user-defined prefix and calculates for each polygon, statistics on pixels that are within.
```
Comment (yjacolin) [inline source/docs/user_manual/plugins/plugins_zonal_statistics.rst]: with an user-defined prefixed

**Comment by DelazJ** [comment]
fixed

**Comment by DelazJ** [comment]
Thanks. I'm sorry :( but I'm afraid you'll have to manually close #588 and #589.

**Pull request merged**